misc/stock_emails.md

```diff
@@ -0,0 +1,111 @@
+
+Mozilla.org is usually the first place a user visits when they want to download Firefox. Because of that, we ask all new communities to begin with localizing the mozilla.org site. Since you have stated that you would like to start a new localization effort, [Peiying] (mailto:pmo@mozilla.com) will be your mentor and point of contact.
+
+Here's how you can get started:
+
+* Create an account on [Pontoon](https://pontoon.mozilla.org/teams).
+* Check our list of existing [L10n teams](https://pontoon.mozilla.org/teams) to see if an effort for your locale already exists. If an effort exists, contact the team manager to join the team and help drive the effort. If it doesn't exist, introduce your new L10n team on the [new locales newsgroup](http://groups.google.com/group/mozilla.dev.l10n.new-locales).
+* Introduce yourself to the L10n community on the [L10n newsgroup](http://groups.google.com/group/mozilla.dev.l10n.new-locales) by subscribing and posting an introduction. In your introduction, please add some brief information about yourself e.g., professional background, interests, etc.) and what you hope to work on (e.g., "I saw that there was xx-XX localization team and I'd like to help them out.").
+* Visit [L10n process doc](https://wiki.mozilla.org/L10n:Localization_Process) to become familiar with how we localize.
+
+For general information on the Mozilla localization program, visit our [wiki home page](https://wiki.mozilla.org/L10n). You're also welcome to read through and post to  [the localization discussion newsgroup](http://groups.google.com/group/mozilla.dev.l10n/topics). 
+Hope to see more from you soon!
+
+##Localizing Thunderbird
+We're glad to see you're interested in localizing Thunderbird. Unfortunately, this is not a project for which Mozilla manages localizations. Please contact [the Thunderbird Council](mailto:thunderbird-council@mozilla.org) to get started.
+
+#Having problems with Pontoon/Pootle
+Sorry to hear that you're having problems with your L10n tool. Please do the following to allow us to troubleshoot your problem(s):
+
+* Search bugzilla to see if a bug has already been filed for your problem(s).
+* Check the newsgroups (specifically [mozilla.dev.l10n](http://groups.google.com/group/mozilla.dev.l10n), [dev.l10n.new-locales](http://groups.google.com/group/mozilla.dev.l10n.new-locales), and [mozilla.tools.l10n](http://groups.google.com/group/mozilla.tools.l10n)) to see if anyone else has had the same problem.
+* If you can't find a discussion thread about your problem, begin a discussion topic in those newsgroups asking for help. Include as much detail as possible.
+* If no one in the discussion group can offer a solution and there are no bugs currently filed, file a new bug providing as many details about the problem as possible.
+
+If the first three steps don't solve your problem and you have to file a bug, rest assured that the Mozilla l10n-drivers will respond promptly.
```

Comment:
We should probably tell them in which component to file the bug? Do we file bugs for Pootle?